app/src/main/java/edu/mills/cs115/fruitthief/database/FruitTreeDao.kt

```diff
@@ -0,0 +1,98 @@
+package edu.mills.cs115.fruitthief.database
+
+import androidx.lifecycle.LiveData
+import androidx.room.*
+
+/**
+ * DAO Database containing two tables, fruit_info_table and tree_info_table.
+ *
+ * @see Tree for tree_info_table
+ * @see Fruit for fruit_info_table
+ */
+@Dao
+interface FruitTreeDAO {
+    /**
+     *
+     * @param tree New tree to be added.
```

Comment:
No period at the end of a `@param` description.

I won't make further comments on the KDoc in this file until you make another pass.